modules/standard/Map.chpl

```diff
@@ -728,7 +728,8 @@ module Map {
     :return: `true` if the contents of two maps are equal.
     :rtype: `bool`
   */
-  proc ==(const ref a: map(?kt, ?vt, ?ps), const ref b: map(kt, vt, ps)): bool {
+  operator map.==(const ref a: map(?kt, ?vt, ?ps),
```

Comment:
I'm not sure what the correct precedent is, but I think list and map should probably match for these operators?